src/Identity/v3/Api.php

```diff
@@ -772,6 +772,7 @@ public function postPolicies(): array
         return [
             'method' => 'POST',
             'path'   => 'policies',
+            'jsonKey' => 'policy',
```

Comment:
@jamiehannaford yes, sure ;) Is it possible to merge this PR and tag a new release? I'm waiting also for #120 ;) Thanks!